rocketpool/api/node/status.go

```diff
@@ -93,7 +86,7 @@ func getStatus(c *cli.Context) (*api.NodeStatusResponse, error) {
 
 	// Get node details
 	wg.Go(func() error {
-		details, err := node.GetNodeDetails(rp, nodeAccount.Address, response.IsHoustonDeployed, nil)
+		details, err := node.GetNodeDetails(rp, nodeAccount.Address, true, nil)
```

Comment:
I'd prefer to see this argument deleted in rocketpool-go first, and then this PR should have a single commit that updates go.mod/sum and removes calls that include this arg

Reply:
https://github.com/rocket-pool/rocketpool-go/pull/26